### PR TITLE
flixhq scraping improved

### PR DESCRIPTION
**Pull request by frost768**
FlixHQ supported media types are movies and series and it also provides subtitles. These were not included in the result of the scraping. This pull request adds them into the result object.

### CODE REVIEW

File: src/backend/helpers/streams.ts [hunk 1]
```diff
@@ -10,9 +10,11 @@ export enum MWCaptionType {
 
 export enum MWStreamQuality {
   Q360P = "360p",
+  Q540P = "540p",
   Q480P = "480p",
   Q720P = "720p",
   Q1080P = "1080p",
+  QAUTO = "auto",
```
Comment (mrjvs) [inline src/backend/helpers/streams.ts:17]: why was QAuto added? it doesnt seem to be used

**Review by mrjvs — CHANGES_REQUESTED** [review]

**Comment by mrjvs** [comment]
tested, works great. just resolve the one review comment and it can be merged

File: src/backend/helpers/streams.ts (hunk 1 above)
Comment (frost768) [reply src/backend/helpers/streams.ts:17]: it does come from flixhq side so if we ever let users to choose their own preferred quality it is just an another option we can display to them

Comment (frost768) [reply src/backend/helpers/streams.ts]: ok, removed it.

**Review by mrjvs — APPROVED** [review]

**Pull request merged**